aiida/tools/importexport/dbexport/__init__.py

```diff
@@ -425,22 +435,35 @@ def export_tree(
         'node_extras': node_extras,
         'export_data': export_data,
         'links_uuid': links_uuid,
-        'groups_uuid': groups_uuid,
+        'groups_uuid': groups_uuid
     }
 
-    # N.B. We're really calling zipfolder.open
+    # N.B. We're really calling zipfolder.open (if exporting a zipfile)
     with folder.open('data.json', mode='w') as fhandle:
         # fhandle.write(json.dumps(data, cls=UUIDEncoder))
         fhandle.write(json.dumps(data))
 
     # Add proper signature to unique identifiers & all_fields_info
     # Ignore if a key doesn't exist in any of the two dictionaries
 
+    license_info = {'allowed_licenses': allowed_licenses, 'forbidden_licenses': forbidden_licenses}
+    for name, value in license_info.items():
+        if value and not isinstance(value, list):
+            # The value is a function (can either be a function, a list or None)
+            license_info[name] = 'Function {} was used to check for {}'.format(value, name.replace('_', ' '))
+
     metadata = {
         'aiida_version': get_version(),
         'export_version': EXPORT_VERSION,
         'all_fields_info': all_fields_info,
         'unique_identifiers': unique_identifiers,
+        'export_parameters': {
+            'link_follow_rules': link_follow_rules,
+            'original_entities': original_entities,
```

Comment:
Maybe `entity_starting_set` would be a bit clearer instead of `original_entities`?